src/platform_impl/ios/view.rs

```diff
@@ -134,6 +134,18 @@ unsafe fn get_view_class(root_view_class: &'static Class) -> &'static Class {
                     height: screen_frame.size.height as f64,
                 }
                 .to_physical(scale_factor.into());
+
+                // If the app is started in landscape, the view frame and window bounds can be mismatched.
+                // The view frame will be in portrait and the window bounds in landscape. So apply the
+                // window bounds to the view frame to make it consistent.
+                let view_frame: CGRect = msg_send![object, frame];
+                let window_bounds: CGRect = msg_send![window, bounds];
+                if view_frame.size.width != window_bounds.size.width
+                    || view_frame.size.height != window_bounds.size.height
+                {
+                    let () = msg_send!(object, setFrame: window_bounds);
```

Comment:
```suggestion
                    let () = msg_send![object, setFrame:window_bounds];
```
I admit that parentheses look cuter than square brackets, but square brackets are the convention we've been using for this macro.

(The use of spaces after colons in `msg_send` has creeped into the codebase, but the Apple documentation doesn't use a space after the colon, so I'm enforcing that.)

Reply:
I didn't even notice I had it different, fixed. 👍

Reply:
I've actually changed this back to have the space because the PR build failed. I ran `cargo fmt` and re-pushed.